beep_crate/src/structures/Tone.mjs

```diff
@@ -0,0 +1,87 @@
+/**
+ * Class Tone
+ * This class defines one frequency or Tone, which can be played unitBlocks.size() times
+ * use: import Tone from '<path_to_this>.mjs'
+ */
+
+export default class Tone{
+    #name               // name of tone (e.g. 'c1', 'd1',...)
+    #frequency;         // Frequency in Hz
+    #unitBlocks;        // Map of when will this tone be played - key is index (natural number) and value is length (natural number)
```

Comment:
Are must index be a natural number or an integer. I would prefer the term 'positive integer'.